src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -17,11 +17,11 @@
   private PersistentStorageAgent mockPersistentStorageAgent;
 
   private final User USER_ONE =
-      new User(UUID.randomUUID(), "test_username_one", Instant.ofEpochMilli(1000), "test_password_one");
+      new User(UUID.randomUUID(), "test_username_one", Instant.ofEpochMilli(1000), "test_password_one", "test_about_one");
```

Comment:
Can we make this a file-level named constant? Same for all the other places you did this

Reply:
Is it better convention to make Strings like these into constants if they are only used once? I changed it in other test classes, but in UserStoreTest, test input like `"test_about_one"` are only used once.